atlp-rwanda/eagles-phantom-backend
# #3RYihb3M Sequelizing Postgre database

**What does this PR do?**
-Postgre Database configuration

**Description of tasks to be completed**
-installing sequelize package
-initialize package
-create postgre database using terminal
-generate sequelize table attribute for Users table
-migrate Table 
-create seeding
-check if users has added using terminal 

**How should be manually tested?**
-install sequelize package` npm install sequelize sequelize-cli pg pg-hstore`
-create database
-create .env file add databse connection string
-create seeding 
-check if user added using terminal

**Relavant trello story**
https://trello.com/c/3RYihb3M/17-configure-the-app-to-use-postgres-database

![database](https://user-images.githubusercontent.com/42033331/94127937-cf331f80-fe59-11ea-8c74-45dbd505535f.jpg)
